bin/make/frank-plugin.sh

```diff
@@ -95,6 +101,7 @@ xcrun xcodebuild build \
   ARCHS="i386 x86_64" \
   VALID_ARCHS="i386 x86_64" \
   ONLY_ACTIVE_ARCH=NO \
+  EFFECTIVE_PLATFORM_NAME="-iphonesimulator" \
```

Comment:
In case it was not clear, we could not build the dylibs or frank libraries with Xcode 7.2.